core/src/test/scala/unit/kafka/log/LogConfigTest.scala

```diff
@@ -94,6 +94,18 @@ class LogConfigTest {
       case TopicConfig.REMOTE_LOG_STORAGE_ENABLE_CONFIG => assertPropertyInvalid(name, "not_a_boolean")
       case TopicConfig.LOCAL_LOG_RETENTION_MS_CONFIG => assertPropertyInvalid(name, "not_a_number", "-3")
       case TopicConfig.LOCAL_LOG_RETENTION_BYTES_CONFIG => assertPropertyInvalid(name, "not_a_number", "-3")
+      case LogConfig.LOG_DIR_PROP => assert(true)
+      case LogConfig.LOG_DIRS_PROP => assert(true)
```

Comment:
What are we testing here?

Reply:
These are both optional strings, that's why I skip validation here.

Reply:
OK, I see. I think we can put all these 3 configs at the same place, and then do nothing instead of assert(true). ex:

`case  LogConfig.LOG_DIR_PROP |  LogConfig.LOG_DIRS_PROP | LogConfig.METADATA_LOG_DIR_PROP => // do nothing because the validation is not happening in LogConfig.